css/_project-custom.scss

```diff
@@ -162,7 +162,7 @@ body {
   li > button {
     @include button-unstyled;
     @include padding(0.7rem 1rem 0.7rem 1.6rem);
```

Comment:
For now, I'm going to avoid comments around implementing v2 tokens — but I can take a pass at all that in a subsequent PR

Reply:
Sounds good, feel free to have at it. The priority here was to make sure it built so it can be improved.